code/parse/sexp.cpp

```diff
@@ -30337,14 +30337,21 @@ void sexp_modify_variable(const char *text, int index, bool sexp_callback)
 		SCP_string temp_text = text;
 		sexp_replace_variable_names_with_values(temp_text);
 
+		if (temp_text.length() > TOKEN_LENGTH - 1)
```

Comment:
Optional: Instead of using `TOKEN_LENGTH - 1` repeatedly, consider adding a line before this `if`-block like
```
const size_t maxCopyLen = TOKEN_LENGTH - 1;
```
and then use `maxCopyLen` everywhere instead.

That'll avoid the issue of one of the `- 1`s getting inadvertently dropped in the future, and it improves readability a bit.

Reply:
Ok